website/docs/r/hci_cluster.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_hci_cluster"
+description: |-
+  Manages an Azure Stack HCI Cluster.
+---
+
+# azurerm_hci_cluster
+
+Manages an Azure Stack HCI Cluster.
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_hci_cluster" "example" {
+  name                = "example-cluster"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  client_id           = data.azurerm_client_config.current.client_id
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+
+  tags = {
+    ENV = "Prod"
+  }
```

Comment:
tags aren't a required field, can we remove these from the example?

Reply:
updated